src/common/types/chainner-scope.ts

```diff
@@ -106,6 +113,18 @@ def convenientUpscale(model: PyTorchModel | NcnnNetwork | OnnxModel, image: Imag
     }
 }
 
+def removeBackground(model: OnnxRemBgModel, image: Image) {
+    Image {
+        width: image.width,
+        height: if model.arch == "u2net_cloth" {
+            image.height * 3
+        } else {
+            image.height
+        },
```

Comment:
I don't like this. Why does the type has to know about arch-specific things? Instead, I think we should give the type scales for width and height. This moves the arch-specific knowledge into the backend.

Reply:
You mean add a `output_height_scale` attribute to the backend classes?

Reply:
Yes, but also to the type. Right now, the knowledge of how u2net_cloth models work is spread between the backend and this function. So I am suggesting to fix that by given the type `scaleWidth: uint` and `scaleHeight: uint` fields. Yes, this means that we need to track those values in the backend and broadcast them over.